bin/src/command_line.dart

```diff
@@ -1,17 +1,47 @@
 library cmdpart;
 
 import 'dart:async';
+import 'dart:io' show FileSystemEntity;
 import "dart:math" show max;
 
 import "package:args/args.dart";
+import 'package:path/path.dart' as p;
+import 'package:logging/logging.dart';
 
 export "package:args/args.dart";
 
+final Logger _log = new Logger("dart_coveralls");
+
 abstract class CommandLinePart {
   final ArgParser parser;
 
   CommandLinePart(this.parser);
 
+  String handlePackagesArg(ArgResults res) {
+    String packagesPath = res["packages"];
+    if (p.isRelative(packagesPath)) {
+      packagesPath = p.absolute(packagesPath);
+    }
+
+    if (!FileSystemEntity.isFileSync(packagesPath)) {
+      print("Packages file does not exist");
+      return null;
+    }
+
+    _log.info(() => "Packages file is ${packagesPath}");
+
+    return packagesPath;
+  }
+
+  static ArgParser addCommonOptions(ArgParser parser) {
+    return parser
+      ..addFlag("help", help: "Displays this help", negatable: false)
+      ..addOption("packages",
```

Comment:
I think it's a reasonable to expect maintained Dart packages to update and for unmaintained ones to break at least given how many other parts of the ecosystem now expect packages over package-root.

But again, I'm probably not the right person to own/maintain this package over the long term. :)  And IMO owners get to decide.  So if you'd like to support both I certainly won't discourage you.